pkg/stores/partition/store.go

```diff
@@ -389,7 +389,7 @@ func ToAPIEvent(apiOp *types.APIRequest, schema *types.APISchema, event watch.Ev
 
 	if event.Type == watch.Error {
 		status, _ := event.Object.(*metav1.Status)
-		apiEvent.Error = fmt.Errorf(status.Message)
+		apiEvent.Error = fmt.Errorf("%s", status.Message)
```

Comment:
Sorry, the `fmt.Errorf("%s", status.Message)` should be `errors.New(status.Message)` because no formatting is necessary.

Reply:
Ok, just fixed it. Thanks!